omod/src/test/java/org/openmrs/BaseWebControllerTest.java

```diff
@@ -0,0 +1,176 @@
+//package org.openmrs;
```

Comment:
@kavitha-sundararajan 
why we are commenting the class instead of removing ?

Reply:
We missed this, it is not needed, removed it now